docs/toolbox/cluster.rst

```diff
@@ -7,22 +7,22 @@ Cluster
 Cluster Scale
 =============
 
-* Set number of nodes with given instance type on AWS
+* Set number of nodes with given instance type
 
 .. code-block:: shell
 
-    ./toolbox/cluster/set_scale.sh <machine-type> <replicas>
+    ./run.py cluster set_scale <machine-type> <replicas>
```

Comment:
maybe this command could receive an AWS flag (and later on some other cloud flags), eg,:
```
     ./run.py cluster set_scale <replicas> --aws-machine-type=<aws-machine-type> 
```

Reply:
The user knows they're running on AWS, they're responsible for passing an AWS machine type as the machine-type parameter. If they know they're running as GCP, they'll pass a GCP machine-type as parameter. I don't see why the flag name itself should encode which cloud we're using

Reply:
> Or maybe I misunderstood your intention

is there nothing at all specific to AWS in these roles? if yes then forget about this thread, I though this part was AWS specific, but if the roles would work as-is with GCP, just with another machine-type, then great :)

Reply:
Even if there is something AWS specific, it should be automatically detected at the role rather than at the CLI level

Reply:
hum, indeed, the role can learn it from OpenShift, and the CLI can stay untouch, I missed that alternative ;)